roles/cobbler/tasks/fetch_cm_repos.yml

```diff
@@ -8,3 +8,9 @@
   git: repo={{ secrets_repo.url }}
        dest=/root/{{ secrets_repo.name }}
        accept_hostkey=true
+
+- name: Symlink /etc/ansible/hosts
+  file: src=/root/{{ secrets_repo.name }}/ansible/inventory/ dest=/etc/ansible/hosts state=link force=yes
```

Comment:
nitpick: can we put src, dest, state and force each on their own lines?

I find that format much easier to read and it's makes more friendly diffs when changing a task that's formatted that way.